### DISCUSSION TITLE
PB CONNEXION TORREFACTEUR-ARTISAN

### DISCUSSION THREAD

**Discussion by Lilidelabastide** [artisan-roaster-scope/artisan#1524] [General]
Bonjour,
Artisan ne reconnait pas mon torréfacteur, je n'arrive pas à le connecter.
Il s'agit d'un GARANTI 12kg

**Comment by MAKOMO** [comment]
Did you install the [CP210x serial driver](https://www.silabs.com/products/development-tools/software/usb-to-uart-bridge-vcp-drivers) on your computer as suggested by the [Artisan Garanti machine page](https://artisan-scope.org/machines/garanti/)?

**Reply by Lilidelabastide** [reply]
oui mais ça ne change rien.

**Comment by Lilidelabastide** [comment]
les températures ne remontent pas ...

**Comment by MAKOMO** [comment]
Did you configure the correct MODBUS port for your USB converter?

Disconnect the USB from your laptop. Open the MODBUS tab (menu Config >> Port, 3rd tab "MODBUS"). Check the ports listed in the "Comm Port" popup. Now connect the USB from your roaster to your laptop again, turn your roasting machine on such that this USB converter is powered and check that same popup again. It should list one additional port. This port is the one to communicate to your roasting machine. Select it and check if Artisan gets readings on turning it ON.

If you do not see any additional comm port while the USB is connected vs disconnected either your USB cable is broken, your serial driver is not correctly installed, or the USB converter of your roaster is broken.
